packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -729,6 +729,7 @@ class _RenderRangeSlider extends RenderBox {
   HorizontalDragGestureRecognizer _drag;
   TapGestureRecognizer _tap;
   bool _active = false;
+  RangeValues _newValues;
```

Comment:
Is there actually a need for storing this as a property? If so, do you need to reset it anywhere?

Reply:
Its so that the values can be shared between the _startInteraction, _handleDragUpdate, and _endInteraction. They can be set in either _startInteraction or _handleDragUpdate, and _endInteraction needs to emit them on onChangeEnd.

It already gets reset whenever there is a new thumb selection, and is only used when the thumb selection is non null